components/gfx/font_cache_thread.rs

```diff
@@ -216,7 +216,8 @@ impl FontCache {
                     url: url.clone(),
                     type_: RequestType::Font,
                     destination: Destination::Font,
-                    origin: url.clone(),
+                    // TODO: Add a proper origin - Can't import GlobalScope from gfx
```

Comment:
I _think_ I can just create an opaque origin out of nothing because it's a font loader anyway, and it's going to be fetching from different places which will never be the same origin.